lib/rabbitmq/cli/ctl/commands/environment_command.ex

```diff
@@ -18,19 +18,22 @@ defmodule RabbitMQ.CLI.Ctl.Commands.EnvironmentCommand do
   @behaviour RabbitMQ.CLI.CommandBehaviour
   use RabbitMQ.CLI.DefaultOutput
 
+  def formatter(), do: RabbitMQ.CLI.Formatters.Erlang
+
+  def scopes(), do: [:ctl, :diagnostics]
+
+  def merge_defaults(args, opts), do: {args, opts}
+
   def validate([_|_], _), do: {:validation_failure, :too_many_args}
   def validate(_, _), do: :ok
-  def merge_defaults(args, opts), do: {args, opts}
 
-  def formatter(), do: RabbitMQ.CLI.Formatters.Erlang
+  use RabbitMQ.CLI.Core.RequiresRabbitAppRunning
```

Comment:
Technically it doesn't but if you stop the app, it's environment won't be printed. I'd expect 99% of users use this command to get that, not e.g. effective `ssl` settings.